karma.config.js

```diff
@@ -5,8 +5,9 @@
 const globby = require('globby');
 const istanbul = require('rollup-plugin-istanbul');
 const { nodeResolve } = require('@rollup/plugin-node-resolve');
-
+// eslint-disable-next-line import/newline-after-import
 process.env.CHROME_BIN = require('puppeteer').executablePath();
+process.env.NODE_ENV = 'test';
```

Comment:
where is this being used?

Reply:
This makes the environment var consistent across jest and karma. I was seeing karma use dev builds.